ofl/sankofadisplay/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1 @@
+N/A
```

Comment:
@n8willis Do we have a description for the family?

Reply:
Added. This is in the DESCRIPTION.en_us.html from upstream; not sure why it wasn't here.